packages/client/src/components/pages/editQuestionPage/EditQuestionPage.tsx

```diff
@@ -26,6 +26,7 @@ const EditQuestionPage: React.FC<IEditQuestionPageProps> = ({
   const [selectedFields, setSelectedFields] = useState<string[] | any[]>([]);
   const [showMsg, setShowMsg] = useState(false);
   const [question, setQuestion] = useState(newQuestion);
+  const [showId, setShowId] = useState("noid");
```

Comment:
```suggestion
  const [show, setShow] = useState(false);
```
Why do you need id all the time? Doesn't that seem unnecessary?